### ISSUE TITLE
Missing styles for first and last child in v0.5 alpha

### ISSUE THREAD

**Issue by silvenon** [tailwindlabs/tailwindcss-typography#209]
### What version of @tailwindcss/typography are you using?

v0.5.0-alpha.1

### What version of Node.js are you using?

v16.10.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

https://github.com/silvenon/repro/tree/tailwind-typography-alpha-margins

### Describe your issue

In v0.4 immediate `:first-child` and `:last-child` of `.prose` didn't have top and bottom margin respectively. This is no longer true in v0.5 alpha even though the code is still there. https://github.com/tailwindlabs/tailwindcss-typography/blob/ddac36a9df3f6bdf70388591ef5837351d41a7ca/src/styles.js#L359-L366

This doesn't appear to be a JIT problem because adding these styles in my stylesheet works, they just have no effect in the `typography` configuration.

**Comment by adamwathan** [comment]
Good catch, looked into this briefly, the problem is these selectors are invalid:

```css
    .prose :where(> :first-child):not(:where([class~=\\"not-prose\\"] *)) {
        margin-top: 0;
    }
    .prose :where(> :last-child):not(:where([class~=\\"not-prose\\"] *)) {
        margin-bottom: 0;
    }
```

You can't put `>` directly inside of a `:where` like that. Need to get that to look something like this instead:

```css
    .prose :where(.prose > :first-child):not(:where([class~=\\"not-prose\\"] *)) {
        margin-top: 0;
    }
    .prose :where(.prose > :last-child):not(:where([class~=\\"not-prose\\"] *)) {
        margin-bottom: 0;
    }
```

Will try get this fixed on Monday.

**Comment by adamwathan** [comment]
Should be fixed in 0.5.0-alpha.2 🤙🏻

**Comment by silvenon** [comment]
So fast! Thanks! ❤️
